forest/src/cli/chain_cmd.rs

```diff
@@ -32,9 +37,15 @@ pub enum ChainCommands {
         /// Include old messages
         #[structopt(short, long)]
         include_old_messages: bool,
-        /// Snapshot output path. Default to `forest_snapshot_{year}-{month}-{day}_height_{height}.car`
-        #[structopt(short, long)]
-        output_path: Option<String>,
+        /// Snapshot output path. Default to forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car
```

Comment:
If we keep `month` in letters and not in `YYYY-MM-DD` format, I would put `_height_{height}` before in the default output path so they are displayed nicely when sorted alphabetical order (ie on DO Spaces).

What about support for `HH-MM-SS` if user want it?

Reply:
I think it's an oversight that the month is not numerical. Should follow the ISO 8601 date format by default.